project/frontend/src/components/PreferenceForm.js

```diff
@@ -52,7 +52,10 @@ async function getLocalCuisines(currLocation) {
  */
 function PreferenceForm(props) {
   const { currLocation, locationName } = props;
-
+  const currHour = parseInt(
+    new Date(Date.now()).toTimeString().substring(0, 2)
```

Comment:
if it's 22:30 would currHour be 22 or 21? just asking because we wouldn't want openNow to be checked at 22:30

Reply:
Oh good point-- it should be [10,22) not [10,22]